src/status_im2/contexts/onboarding/create_profile/view.cljs

```diff
@@ -0,0 +1,38 @@
+(ns status-im2.contexts.onboarding.create-profile.view
+  (:require [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [status-im2.contexts.onboarding.create-profile.style :as style]
+            [utils.i18n :as i18n]
+            [status-im2.contexts.onboarding.common.background.view :as background]
+            [utils.re-frame :as rf]))
+
+(defn navigation-bar
+  []
+  [rn/view {:style style/navigation-bar}
+   [quo/page-nav
+    {:align-mid?   true
+     :mid-section  {:type :text-only :main-text ""}
```

Comment:
I wonder if this empty string is actually needed, I feel it should be an optional parameter and `main-text` should have an empty string as its default value